test/lib/diff.js

```diff
@@ -0,0 +1,1095 @@
+const { resolve } = require('path')
+const t = require('tap')
+const requireInject = require('require-inject')
+
+const noop = () => null
+let libnpmdiff = noop
+let rlp = () => 'foo'
+const defaultFlatOptions = {
+  defaultTag: 'latest',
+  diff: null,
```

Comment:
If it's `[String, Array]`, then the default will be `[]`, which is probably more useful :)